src/img/icons.json

```diff
@@ -86,6 +86,7 @@
     "return-arrow",
     "search",
     "settings",
+    "share",
```

Comment:
This looks more like upload rather than share. But at any rate, we need more descriptive naming for these

Reply:
For the share icon, this was added in another PR recently: https://github.com/JoinColony/colonyDapp/blob/9ebbe39d5c0649ec6e60d2d5cf045339c22b4882/src/img/icons/share.svg
And I will try to pay more attention to naming.